sklearn/utils/_testing.py

```diff
@@ -452,6 +457,10 @@ def set_random_state(estimator, random_state=0):
         not joblib.parallel.mp, reason="joblib is in serial mode"
     )
 
+    fails_if_unstable_openblas = pytest.mark.xfail(
```

Comment:
not sure if xfail is a good idea for code that can segfault. Maybe skip is a better idea.

Reply:
This was marked resolved without being addressed or commented on.

Reply:
My mistake multitasking and forgetting to take your suggestion in the patch of this commit.